src/csharp/SDK/Transformation.cs

```diff
@@ -142,6 +142,129 @@ public void DepthImageToColorCamera(Image depth, Image transformed)
             }
         }
 
+        /// <summary>
+        /// Transforms an Image and an Custom Image from the depth camera perspective to the color camera perspective.
```

Comment:
/// Transforms depth Image and a custom Image from the depth camera perspective to the color camera perspective.

Reply:
Fixed! Thanks,